Billmike/Chessecake-Factory
# Setup Backend dependencies

### What does this PR do?
Setups backend dependencies for the project
### Description of tasks to be completed?
- Setup express, morgan, sequelize, body-parsesr and PostgreSQL database for the application
### How should this be tested?
- After cloning to local machine, ```cd Chesscake-Factory```
- run ```npm install``` to install dependencies on local machine
- run ```npm start``` to start development server and point browser to localhost:8000 to see the message "Welcome to nothingness"
### Screesnshots
![pr1](https://user-images.githubusercontent.com/24798364/30584716-7fa2733a-9d22-11e7-87eb-5addf363e7f1.PNG)
